Plots/Overlays/NCL_overlay_6.py

```diff
@@ -0,0 +1,151 @@
+    true representation of the data, as the magnitude of the majority of wind
+    vectors are between 3 and 6 units with only a handful being greater than 13
+    and only one near 20. Because of this, we have chosen not to manipulate the
+    vector glyphs to appear more uniform as this would poorly represent the
+    data and be misleading. This is why the reference vector in this Python
+    example is longer than the NCL example and why the lengths vary more
+    between the minimum and maximum wind speeds.
+
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import matplotlib.colors as mcolors
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+uf = xr.open_dataset(gdf.get("netcdf_files/Ustorm.cdf"))
+vf = xr.open_dataset(gdf.get("netcdf_files/Vstorm.cdf"))
+pf = xr.open_dataset(gdf.get("netcdf_files/Pstorm.cdf"))
+tf = xr.open_dataset(gdf.get("netcdf_files/Tstorm.cdf"))
+u500f = xr.open_dataset(gdf.get("netcdf_files/U500storm.cdf"))
+v500f = xr.open_dataset(gdf.get("netcdf_files/V500storm.cdf"))
+
+p = pf.p.isel(timestep=0).drop('timestep')
+t = tf.t.isel(timestep=0).drop('timestep')
+u = uf.u.isel(timestep=0).drop('timestep')
+v = vf.v.isel(timestep=0).drop('timestep')
+u500 = u500f.u.isel(timestep=0).drop('timestep')
+v500 = v500f.v.isel(timestep=0).drop('timestep')
+time = vf.timestep
+
+# Convert Pa to hPa
+p = p/100
+# Convert K to F
+t = (t - 273.15) * 9/5 + 32
+
+###############################################################################
+# Create map:
+fig = plt.figure(figsize=(8, 7))
+proj = ccrs.LambertAzimuthalEqualArea(central_longitude=-100,
+                                      central_latitude=40)
+
+# Set axis projection
+ax = plt.axes([0, 0.2, 0.8, 0.7], projection=proj)
+# Create inset axes for color bars
+cax1 = inset_axes(ax, width='5%', height='100%', loc='lower right',
+                  bbox_to_anchor=(0.125, 0, 1, 1),
+                  bbox_transform=ax.transAxes,
+                  borderpad=0)
+
+cax2 = inset_axes(ax, width='100%', height='7%', loc='lower left',
+                  bbox_to_anchor=(0, -0.15, 1, 1),
+                  bbox_transform=ax.transAxes,
+                  borderpad=0)
+
+# Set extent to include roughly the United States
+ax.set_extent((-128, -58, 18, 65), crs=ccrs.PlateCarree())
+
+# Add map features
+transparent = (0, 0, 0, 0)  # RGBA value for a transparent color for lakes
+ax.add_feature(cfeature.OCEAN, color='lightskyblue', zorder=0)
```

Comment:
Can you add a comment about what the zorder keyword does?

Reply:
While piecing this example together, I found that I needed to use `zorder` to correctly layer the different types of plots. Now it is no longer needed, so I have removed all uses of it.